src/content/ko/fundamentals/design-and-ui/animations/animations-and-performance.md

```diff
@@ -24,13 +24,13 @@ description: 애니메이션은 성능이 우수해야 합니다. 그렇지 않
 
 레이아웃 또는 그림 그리기를 트리거하는 애니메이션 속성은 가급적 피해야 합니다. 최신 브라우저의 경우 이는 애니메이션을 `opacity` 또는 `transform`으로 제한하는 것을 의미하며 이 둘은 브라우저에서 효과적으로 최적화될 수 있습니다. 애니메이션이 JavaScript 또는 CSS에 의해 처리되는지 여부는 상관이 없습니다.
 
-개별 CSS 속성에 의해 트리거되는 작업의 전체 목록은 [CSS 트리거](http://csstriggers.com)을 참조하십시오. [HTML5 Rocks에서 고성능 애니메이션](http://www.html5rocks.com/en/tutorials/speed/high-performance-animations/) 생성에 대한 전체 가이드도 참조하십시오.
+개별 CSS 속성에 의해 트리거되는 작업의 전체 목록은 [CSS 트리거](http://csstriggers.com){: .external }을 참조하십시오. [HTML5 Rocks에서 고성능 애니메이션](http://www.html5rocks.com/en/tutorials/speed/high-performance-animations/) 생성에 대한 전체 가이드도 참조하십시오.
```

Comment:
I believe you need a space after the external on the first line, and the second one on this line isn't tagged as external